RNAdam-core/src/main/scala/org/bdgenomics/RNAdam/algorithms/quantification/Tare.scala

```diff
@@ -132,4 +134,56 @@ object Tare extends Serializable {
 
     calibratedKmers
   }
+
+  /**
+   * Trains a linear regression model of the log of the transcript abundances
+   * versus the log of the transcript lengths.
+   * This is used to recalibrate the transcript abundances.
+   * Warning: This procedure assumes that the abundances are all > 0
+   * The log is undefined for numbers <= 0
+   *
+   * @param muHat An RDD containing the transcript abundances.
+   * @param tLen A mapping from transcript names to their lengths.
+   * @return The recalibrated transcript abundances.
+   */
+  def calibrateTxLenBias(muHat: RDD[(String, Double, Iterable[Long])],
+                         tLen: scala.collection.Map[String, Long]): RDD[(String, Double, Iterable[Long])] = {
+    // A local (not distributed) copy of muHat without the equivalence
+    // class ids. Removing these ids saves space since they are
+    // not used here.
+    val local: Array[(String, Double)] = muHat.map(mh => (mh._1, mh._2)).collect()
+
+    // Calculate the log of the average (mean) abundance.
+    // Since all the abundances sum to 1.0 by definition,
+    // this is simply 1.0 divided by the number of transcripts.
+    // Use the fact that log(1.0 / x) = -log(x)
+    val mean: Double = -log(local.length.toDouble)
+
+    // Do linear regression by constructing and solving matrix equation.
```

Comment:
Can you add a note here to explain that we didn't use MLLib because of convergence problems?